### PR TITLE
New ARC-0013

### PR DESCRIPTION
**Pull request by scholtz**
I request comment for ARC-0013 - Community Algorand Standards

### CODE REVIEW

File: ARCs/arc-0013.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+Living - A special status for standard that are designed to be continually updated and not reach a state of finality.
+
+### Algorand foundation role
+
+Algorand foundation manages list of standards provided by the community and assigns them a standard identifier.
+
+### List of the Community Algorand Standards
+
+List of standards is available here: https://github.com/algorandfoundation/ARCs/blob/main/ARCs/CAS.json
```
Comment (barnjamin) [inline ARCs/arc-0013.md:57]: this link 404s

**Comment by barnjamin** [comment]
This is an interesting idea for having more machine readable ARC documents. 

As a first comment I think I'd prefer the URL point to the spec document _in_ this repo. 

Also maybe a PullRequest type field that points to the actual PR containing the proposal. 

Are you expecting that 2 prs are filed? 1 for the json object and 1 for the spec document?

File: ARCs/arc-0013.md (hunk 1 above)
Comment (scholtz) [reply ARCs/arc-0013.md:57]: it is 404 because the file is in this pull request

**Comment by scholtz** [comment]
lets discuss the topic here please: https://github.com/algorandfoundation/ARCs/issues/20

After this pr will be committed, we will loose the discussion if we discuss it here.

Please note that proposed CAS.json file is included in this PR https://github.com/algorandfoundation/ARCs/pull/19/files

**Comment by barnjamin** [comment]
>After this pr will be committed, we will loose the discussion if we discuss it here.

The conversation will persist in the same was as an issue. It may be closed but will still be visible. 

I'd prefer comments in the PR because any notes on the spec file itself will be inline as well.

**Comment by jeremyv3** [comment]
I'm not clear on the necessity of this proposal at this time.

1) I wasn't able to find any example references that have defined such an ARC schema.  Do you have any example references to share to see what others have done for this? e.g. Ethereum, Python

2) It seems like Fergals original request to define ARC formatting on Algorand should be resolved before a standard JSON schema for ARC submittals is defined so that we know whether there are additional properties that are needed.
https://github.com/algorandfoundation/ARCs/issues/9

3) Where or in what process would this schema be used? When submitting to github? If so where?

4) Is there a need in the community at large for this? How great?

5) Does this proposal mean to suggest by the 'url' property name that the contents of the ARC requests could be kept outside of Algorand's github? I believe ARC's should remain on Algorand's github in all cases for historic purposes.

To me defining an official Algorand schema for this at this time seems premature and rushed before a wider community need for a schema of this type has been defined and established.

**Comment by SudoWeezy** [comment]
Closed in favor of [ARC-0](https://github.com/algorandfoundation/ARCs/blob/main/ARCs/arc-0000.md)

**Pull request closed without merging**